### DISCUSSION TITLE
A set of magical animation libraries for tailwindcss, accessed using the tailwindcss plugin.

### DISCUSSION THREAD

**Discussion by Sylvenas** [tailwindlabs/tailwindcss#7519] [Show and tell]
GitHub repository [tailwindcss-magic](https://github.com/Sylvenas/tailwindcss-magic)

A set of magical animation libraries for [tailwindcss](https://tailwindcss.com/), accessed using the [tailwindcss plugin](https://tailwindcss.com/docs/plugins).

Preview the example live on CodeSandbox: [Open in CodeSandbox](https://codesandbox.io/s/fancy-glade-3197y8?file=/pages/index.js)

> How to customize the animation properties？ Try [tailwindcss-animation-property](https://github.com/Sylvenas/tailwindcss-animation-property).

<div style="text-align:center" align="center">
  <img src="https://p1.music.126.net/T4JA1NmZlSTZZZK4EgBncA==/109951164699178486.gif" />
</div>

**Comment by alonronin** [comment]
very nice!

**Reply by Sylvenas** [reply]
Thanks!
